portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,29 @@
+/*
+This will make text still reflect the default font size
+of the user
+*/
+body {
+    font-size: 1.6rem;
+}
+
+/* 
+Since most browsers have a default of 16px, 1rem = 10px
+Unless if the user increased their default font size
+*/
+html {
+    font-size: 62.5%;
+}
+
+.img-primary {
+    width: 100%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 65rem;
 }
 
 #greeting-container {
-  margin-top: 20px;
-}
+  margin-top: 2rem;
+}
```

Comment:
It's best practice to leave a single newline at the end of the file. It's not part of the style guide, but there are some good reasons for it:

- If you ever work with the repository on the command line and print a file to the terminal, you'll get a nice clean line break at the end. Otherwise your prompt may continue on the same line.
- Some editors (vim) automatically add a new line to files, so you could get inconsistencies between files.
- Gets rid of the warning in GitHub diffs!

Reply:
Thanks for letting me know - didn't know this existed! All fixed now.